library/Zend/XmlRpc/Value.php

```diff
@@ -486,13 +486,17 @@ protected static function _createSimpleXMLElement(&$xml)
      */
     protected static function _extractTypeAndValue(SimpleXMLElement $xml, &$type, &$value)
     {
-        list($type, $value) = each($xml);
+        $type = key($xml);
+        $value = current($xml);
+        next($xml);
```

Comment:
These lines should mimic the old `list` ... `each` syntax (including moving the array pointer).